CONDUCT.md

```diff
@@ -0,0 +1,111 @@
+# Scikit-Optimize Code of Conduct
+
+Scikit-Optimize is an engaged and respectful community made up of people
+from all over the world. Your involvement helps us to further our
+mission.
+
+The goal of scikit-optimize is to become the go to project for black box
+optimization, freely available to the reader and everyone else. As any other
+open source project, it requires a community of people who maintain and
+contribute to it. The quality of the project depends on the engagement and
+size of the community; thus you as a user of skopt directly benefit from
+creating positive environment that encourages people to contribute to the
+project.
+
+To ensure that we build the best possible environment for our community, we
+introduced this code of conduct. When these guidelines are not followed, it
+can be detrimental to the community, and in turn not beneficial to the users of
+scikit-optimize.
```

Comment:
WDYT? I am not sure I agree with "no damage is done to reputation of people who violate this COC". This is hard to do no? Imagine someone who seriously violates the COC and we decide to ask them to leave. You probably have to announce that in public, which does damage their reputation :-/

Reply:
I am fine with removing the reputation thingy. It seemed also a bit shaky to me when I wrote it.

Reply:
Ah it was also intended to be "by acting according to this CoC, you risk less with your reputation"